src/app/GitUI/CommandsDialogs/FormPush.cs

```diff
@@ -780,24 +780,27 @@ private void BranchSelectedValueChanged(object sender, EventArgs e)
             {
                 if (PushToRemote.Checked)
                 {
-                    if (_NO_TRANSLATE_Branch.SelectedItem is GitRef branch)
+                    IGitRef selectedRef = _NO_TRANSLATE_Branch.SelectedItem as IGitRef;
```

Comment:
Why change the name?
```suggestion
                    IGitRef branch = _NO_TRANSLATE_Branch.SelectedItem as IGitRef;
```
or
```suggestion
                    IGitRef branchRef = _NO_TRANSLATE_Branch.SelectedItem as IGitRef;
```

Reply:
`branch` is not expressive at all for me.
Because we take the `SelectedItem` it seems natural to me to have `selected` in the name.
I would prefer `selectedBranchRef` or, as a 2nd choice, `branchRef`.

Reply:
I, too, am fine with `selectedBranchRef`.